requirements.txt

```diff
@@ -1,4 +1,4 @@
-certifi>=2018.4.16
+certifi>=2023.07.22
```

Comment:
Bumping this is probably considered a major version change since applications that are pinned to a lesser dependency can't actually use this next version without changing their dependencies as well.